docs/fides/docs/installation/configuration.md

```diff
@@ -10,16 +10,27 @@ After initializing fidesctl, a default configuration file will be generated and
 
 ```toml title="fidesctl.toml"
 [api]
+
+# If FIDESCTL_TEST_MODE env var is set to True, the test_database_url
+# will be used instead of the database URL to avoid overwriting production data
+test_database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
+
+# The SQLAlchemy connection string used to connect to the database
+# https://docs.sqlalchemy.org/en/14/core/engines.html#postgresql
 database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-log_destination = ""
-log_level = 20
-log_serialization = ""
+
+# Logging
+log_destination = "" # Also accepts: Any valid file path
+log_level = INFO # Also accepts: TRACE, DEBUG, WARNING, ERROR, CRITICAL
+log_serialization = "" # Also accepts: JSON
 
 [cli]
-server_url = "http://localhost:8080"
+local_mode = False # Tells fidesctl to run without calling a webserver
```

Comment:
I'm struggling with a suggestion here, but I think setting this to `False` and then saying that the option "tells fidesctl to run without calling a webserver" is slightly confusing - it implies that the `False` setting is what achieves this.
```suggestion
local_mode = True # Run fidesctl without any external connections (default: False)
```

Reply:
added a better description, good call